ProiectPractica/Validator/ConfigurationValidator.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Eventing.Reader;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProiectPractica.Validator
+{
+    public class ConfigurationValidator
```

Comment:
Validators are a good example of a static entities, as they generally act as a collection of static validation methods. Please change to static and adjust usages in code.